reactor-netty-http/src/main/java/reactor/netty/http/server/logging/AccessLog.java

```diff
@@ -22,12 +22,18 @@
 
 /**
  * Log the http access information.
+ * All logs are written to a Logger named {@code reactor.netty.http.server.AccessLog} at INFO level.
+ * <p>
+ * See {@link AccessLogFactory} for convenience methods to create an access log factory to be passed to
+ * {@link reactor.netty.http.server.HttpServer#accessLog(AccessLogFactory)} during server configuration.
+ * Note that access logging must be globally enabled first for that configuration to
+ * be taken into account, see the {@link reactor.netty.ReactorNetty#ACCESS_LOG_ENABLED} system property.
```

Comment:
the `@since 1.0.1` can be added there as well @aneveu (class existed before that but was package private)